messages/models.py

```diff
@@ -0,0 +1,36 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+
+
+class Message(models.Model):
+    user = models.ForeignKey(User, on_delete=models.CASCADE)
+    date = models.DateTimeField(default=timezone.now)
+    text = models.TextField()
+    # Message.id field auto-generated by Django, will be used down the road but no need to define it
+    category = models.ManyToManyField(Category, blank=True)
+
+    @classmethod
+    def addMessage(cls, user, category, text):
+        newMessage = Message.objects.create()
+        newMessage.user = user
+        newMessage.category = category
+        newMessage.text = text
+        newMessage.date = timezone.now
+        return newMessage
+
+    @classmethod
+    def deleteMessage(cls, messageID):
+        Message.objects.filter(id=messageID).delete()
+        pass
+
+    def editMessage(self, text):
+        self.text = text
+        pass
+
+
+class Category(models.Model):
+    name = models.TextField(max_length=50)
+    message = models.ManyToManyField(Message, blank=True)
```

Comment:
it looks from Django documentation that ManyToMany is used only in one of the related classes:
https://docs.djangoproject.com/en/3.1/topics/db/examples/many_to_many/

Reply:
That's it!!
When both fields were listed, there was a mutual dependency so that even changing the order of classes in the file wouldn't help. Thanks!